src/display/editor/tools.js

```diff
@@ -1379,8 +1379,10 @@ class AnnotationEditorUIManager {
     });
   }
 
-  translateSelectedEditors(x, y) {
-    this.commitOrRemove();
+  translateSelectedEditors(x, y, commit = true) {
+    if (commit) {
```

Comment:
Personally I've always felt that having `undefined` imply `true` for a default value feels inconsistent, so can we please avoid that as follows instead?
```suggestion
  translateSelectedEditors(x, y, skipCommit = false) {
    if (!skipCommit) {
```